middle_end/flambda2/from_lambda/closure_conversion.ml

```diff
@@ -1755,7 +1757,7 @@ let make_unboxed_function_wrapper acc function_slot ~unarized_params:params
                 ~current_region:my_region))
         Debuginfo.none ~inlined:Inlined_attribute.Default_inlined
         ~inlining_state:(Inlining_state.default ~round:0)
-        ~probe:None ~position:Normal
+        ~probe:None ~tail:Default_tail ~position:Normal
```

Comment:
I don't understand this code well enough to know whether I should propagate some tail annotation from a surrounding scope...

Reply:
I assume that when we introduce the new type we can put Phantom_function here.

Reply:
I think I agree that there's no annotation to propagate here.
1. If this call inherited some tailcallness properties from the boxed version, I'd expect `~position` to not be constant.
2. Dropping annotations only makes the analysis more conservative (we might start flagging edges that we forgot were explicitly labelled `[@nontail]`), so either way I don't think we should worry about this too much for now.

If you're still concerned, perhaps leave a CR?

Reply:
I'm less concerned now, I just wanted to flag it for closer review. Thanks for taking a look.